.ci/install_ovmf_x86_64.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+# Copyright 2022 Advanced Micro Devices, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o errtrace
+
+cidir=$(dirname "$0")
+source "${cidir}/lib.sh"
+
+main() {
+	local buildscript="${katacontainers_repo_dir}/tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh"
```

Comment:
This whole logic can be simplified by calling `build_static_artifact_and_install x86_64-ovmf`. See .ci/install_tdvf.sh and .ci/install_kata_kernel.sh as examples.

Reply:
Changed. Thanks!